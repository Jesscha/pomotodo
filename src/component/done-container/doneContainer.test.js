import React from 'react';
import { shallow } from 'enzyme';
import {DoneContainer} from './doneContainer.component';

describe("<DoneContainer />", ()=>{
    const wrapper = shallow(<DoneContainer doneItems={[]} />)
   


    it("renders DoneContainer", ()=>{
        expect(wrapper).toMatchSnapshot();
    }); 

    

    
    })





