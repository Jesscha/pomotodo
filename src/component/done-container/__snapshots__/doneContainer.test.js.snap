// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DoneContainer /> renders DoneContainer 1`] = `ShallowWrapper {}`;
